core/src/test/scala/integration/kafka/api/LegacyAdminClientTest.scala

```diff
@@ -33,7 +33,7 @@ import org.junit.{After, Before, Test}
 import org.junit.Assert._
 import scala.collection.JavaConverters._
 
-class AdminClientTest extends IntegrationTestHarness with Logging {
+class LegacyAdminClientTest extends IntegrationTestHarness with Logging {
```

Comment:
Good idea.  I added some JavaDoc here and also on the Scala AdminClient with that info